receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -215,3 +220,20 @@ func buildGaugeOrCounterPoint(parsedMetric *statsDMetric, now *timestamppb.Times
 	metricTypeSetter(parsedMetric, isDouble)
 	return point, nil
 }
+
+func buildTimerPoint(parsedMetric *statsDMetric, now *timestamppb.Timestamp, metricTypeSetter func(parsedMetric *statsDMetric)) (*metricspb.Point, error) {
+	var point *metricspb.Point
```

Comment:
Just out of curiosity: do we have this already in OpenTelemetry?

Reply:
Now we are using OpenCensus type, will move to OpenTelemetry type later